inc/Engine/Common/Cache/FilesystemCache.php

```diff
@@ -178,9 +178,6 @@ public function has( $key ) {
 	 */
 	public function generate_url( string $url ): string {
 		$path = $this->generate_path( $url );
-		if ( ! $this->filesystem->exists( $path ) ) {
```

Comment:
@Miraeld is it intended to remove this check? (it seems was here to guard against error)

Reply:
This was an intentional decision.

While addressing the URL problem you mentioned earlier, I encountered another issue. The `url` I received turned out to be a relative path. In other words, it was attempting to load images using URLs like `https://domain.com/var/www/html/wp-content/rocket-test-data/image.jpg`.

To resolve this, I introduced the `transform_relative_to_absolute` function. Its purpose is to convert a relative path to an absolute one. However, constructing an absolute path requires knowledge of the source URL. In my case, the URL doesn't originate from the original CSS file but rather from the cached version. Unfortunately, the function responsible for this process was bailing out prematurely, making it impossible to obtain the URL of the cached version, which is saved at the end of the process.

If the function doesn't bail out, you can retrieve the URL of the cached version CSS. Subsequently, you can build an absolute URL and, consequently, display the image with the correct URL.